kotlinx-coroutines-core/native/src/EventLoop.kt

```diff
@@ -26,5 +26,5 @@ internal class EventLoopImpl: EventLoopImplBase() {
 
 internal actual fun createEventLoop(): EventLoop = EventLoopImpl()
 
-@Suppress("DEPRECATION")
+@Suppress("DEPRECATION_ERROR")
```

Comment:
The change can be merged to develop if we suppress both `DEPRECATION` and `DEPRECATION_ERROR`.
But, it would seem weird.

Reply:
This also underscores the need to refactor the usages of `nanoTime()` to use the Duration API instead. Having the kotlinx-datetime `Clock` and `Instant` in stdlib would simplify this migration.

Reply:
> Can we replace it with the proposed replacement instead?

There is not direct replacement for the `getTime` currently.

Reply:
It might make sense to leave `getTimeX` deprecated with warning until a replacement is occurs, though.

Reply:
Neither `Clock` nor `Instant` would be monotonic, though.

Maybe make the actualization on more granular groups of targets (linux, macos, mingw, androidNative) and invoke OS-level APIs to get the monotonic nano time. They're all just one- or two-liners.

Reply:
I have replaced the K/N `getTimeNanos()` usage with `TimeSource.Monotonic.markNow()`.
Subsequently, now `ComparableTimeMark` is used to designate a point of time instead of `Long`.
PTAL.